### PR TITLE
New tool: biopet-extractadaptersfastqc

### PR DESCRIPTION
**Pull request by rhpvorderman**
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).
***

This pull request was automatically generated by [sbt-bioconda](https://github.com/biopet/sbt-bioconda). A release of [biopet-extractadaptersfastqc](https://github.com/biopet/extractadaptersfastqc) is automatically generated if the `sbt biocondaRelease` command is issued. As part of this process the recipes are tested using CircleCi as described [here](https://bioconda.github.io/contribute-a-recipe.html#test-locally). 

***

### Tool description

ExtractAdaptersFastqc reads which adapter sequences where found from a FastQC raw report. These sequences can be used as input for a QC tool such as cutadapt.

The sequences can be output in plain text format with a newline character as a separator between the sequences. Alternatively the sequences can be output in FASTA format.

For documentation and manuals visit our [github.io page](https://biopet.github.io/extractadaptersfastqc).

### CODE REVIEW

**Comment by rhpvorderman** [comment]
Hi @bioconda/core . Can you review this new recipe? It was automatically generated, so if there are things that need to improve we would like to know. We can then update the automation process.

File: recipes/biopet-extractadaptersfastqc/0.1/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  run:
+  - openjdk >=8,<9
+  - python
+about:
+  home: https://github.com/biopet/extractadaptersfastqc
+  license: MIT
+  summary: ExtractAdaptersFastqc reads which adapter sequences where found from a FastQC raw report.
+  description: |-
+    ExtractAdaptersFastqc reads which adapter sequences where found from a FastQC raw report. These sequences can be used as input for a QC tool such as cutadapt.
+
```
Comment (bgruening) [inline recipes/biopet-extractadaptersfastqc/0.1/meta.yaml]: the newlines are not needed here, isn't it?

File: recipes/biopet-extractadaptersfastqc/0.1/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  commands:
+  - biopet-extractadaptersfastqc --version
+  - biopet-extractadaptersfastqc --help
+extra:
+  notes: 'biopet-extractadaptersfastqc is a Java program that comes with a custom
+    wrapper shell script. By default ''no default java option'' is set in the wrapper.
+    The command that runs the program is ''biopet-extractadaptersfastqc''. If you
+    want to overwrite it you can specify memory options directly after your binaries.
+    If you have _JAVA_OPTIONS set globally this will take precedence. For example
+    run it with ''biopet-extractadaptersfastqc -Xms512m -Xmx1g''. '
```
Comment (bgruening) [inline recipes/biopet-extractadaptersfastqc/0.1/meta.yaml]: addding a DOI and/or a bio.tools identifier?

File: recipes/biopet-extractadaptersfastqc/0.1/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+about:
+  home: https://github.com/biopet/extractadaptersfastqc
+  license: MIT
+  summary: ExtractAdaptersFastqc reads which adapter sequences where found from a FastQC raw report.
+  description: |-
+    ExtractAdaptersFastqc reads which adapter sequences where found from a FastQC raw report. These sequences can be used as input for a QC tool such as cutadapt.
+
+    The sequences can be output in plain text format with a newline character as a separator between the sequences. Alternatively the sequences can be output in FASTA format.
+
+    For documentation and manuals visit our [github.io page](https://biopet.github.io/extractadaptersfastqc).
```
Comment (bgruening) [inline recipes/biopet-extractadaptersfastqc/0.1/meta.yaml]: just the URL, not markdown

File: recipes/biopet-extractadaptersfastqc/0.1/meta.yaml (hunk 2 above)
Comment (rhpvorderman) [reply recipes/biopet-extractadaptersfastqc/0.1/meta.yaml]: This is just a small tool that extracts adapters from a fastqc report. It does not do any analysis. 
It is useful as part of a pipeline that executes fastqc and then runs cutadapt. We used it in the [biopet pipelines](https://github.com/biopet/biopet) in the flexiprep pipeline. But we are now transitioning to [wdl](https://github.com/biowdl/QC) so we are using it in there.

We might publish it as part of publishing the pipeline, but as of now there is no DOI yet.

**Comment by rhpvorderman** [comment]
Thanks @bgruening for reviewing. I changed a few things, I hope it looks okay now.

**Comment by bgruening** [comment]
@rhpvorderman thanks!
Please follow the cb3 migration, your scripts needs to be adopted.

**Pull request merged**